tiles.js

```diff
@@ -0,0 +1,130 @@
+(function(){
+
+  $(document).ready(function(){
+    var Tile = function (){
+        this.inner = '#48895e',
+        this.outer = '#34495e',
+        this.current = this.outer
+        this.height = 200,
+        this.width =  200,
+        this.element = function() {
+            var div = document.createElement("div");
+            $(div).css('background-color', this.current);
+            $(div).css('height', this.height).css('width', this.width).css('margin', 2);
+            return div;
+        };
+    };
+    
+
+    Tile.prototype = {
+      build: function() {
+            var div = document.createElement("div");
+            $(div).css('height', this.height).css('width', this.width).css('margin', 2);
+            $(div).css('background-color', this.current)
+            return div;
+        },
+      toggleColor: function() {
+        if (this.current === this.inner) {
+            this.current = this.outer;
+        } else {
+            this.current = this.inner;
+        }
+      },
+      changeColor: function() {
+        this.toggleColor();
+        return this.build();
+      },
+      delayChangeColor: function() {
+        this.toggleColor();
+        setTimeout('this.changeColor()', 2000);
+      },
+      randomColor: function() {
+        var randomColor = '#'+Math.floor(Math.random()*16777215).toString(16);
+        this.inner = randomColor;
+      }
+    }
+
+    var assignTile = function(cell, div) {
+      $(cell).html(div);
+    }
```

Comment:
Rather than having this free-floating function, I would probably put it on one of the classes.